From b011a57e41ccbd402ecdcb53514ba76b898ec22e Mon Sep 17 00:00:00 2001
From: Jason Wang <wangborong@cdjrlc.com>
Date: Sat, 11 Dec 2021 17:19:14 +0800
Subject: [PATCH] RAS/CEC: Remove a repeated 'an' in a comment
Git-commit: b011a57e41ccbd402ecdcb53514ba76b898ec22e
Patch-mainline: v5.17-rc1
References: bsn#84
Modified-by-SEL: No


The double `an' in the comment in line 41 is repeated. Remove one of
them.

Signed-off-by: Jason Wang <wangborong@cdjrlc.com>
Signed-off-by: Borislav Petkov <bp@suse.de>
Link: https://lore.kernel.org/r/20211211091914.268823-1-wangborong@cdjrlc.com
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 drivers/ras/cec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/ras/cec.c b/drivers/ras/cec.c
index d7894f178bd4..42f2fc0bc8a9 100644
--- a/drivers/ras/cec.c
+++ b/drivers/ras/cec.c
@@ -38,7 +38,7 @@
  * elements entered into the array, during which, we're decaying all elements.
  * If, after decay, an element gets inserted again, its generation is set to 11b
  * to make sure it has higher numerical count than other, older elements and
- * thus emulate an an LRU-like behavior when deleting elements to free up space
+ * thus emulate an LRU-like behavior when deleting elements to free up space
  * in the page.
  *
  * When an element reaches it's max count of action_threshold, we try to poison
-- 
2.26.2

